Library/Homebrew/cmd/style.rb

```diff
@@ -1,3 +1,25 @@
+#:  * `style` [`--fix`] [`--display-cop-names`] [<formulae>|<files>]:
+#:    Check formulae or files for conformance to Homebrew style guidelines.
+#:
+#:    <formulae> is a list of formula names.
+#:
+#:    <files> is a list of file names.
+#:
+#:    <formulae> and <files> may not be combined. If both are omitted, style will run
+#:    style checks on the whole Homebrew `Library`, including core code and all
+#:    formulae.
+#:
+#:    If `--fix` is passed and `$HOMEBREW_DEVELOPER` is set, style violations
```

Comment:
I don't think we include the dollar sign in other parts of our man page when referring to environment variables.